bin/brew

```diff
@@ -44,4 +44,17 @@ fi
 
 HOMEBREW_LIBRARY="$HOMEBREW_REPOSITORY/Library"
 
-source "$HOMEBREW_LIBRARY/Homebrew/brew.sh"
+if [[ -n "$HOMEBREW_ENV_FILTERING" ]]
+then
+  PATH="/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
Not really necessary. Superenv does its only PATH cleansing; stdenv^ and `env :userpaths`^^ would be broken by this.

^ Someone will shout at me saying stdenv is deprecated, but it apparently still exists as far as I can tell, although I personally don't use it and am not rooting for it.
^^ Also not popular.